deploy/kubernetes/README.md

```diff
@@ -5,7 +5,44 @@
 This is the list of officially supported Elastic Agents, with example manifests to run
 them in Kubernetes:
 
-Beat | Description
+Agent Mode | Description
 ---- | ----
 [Elastic Agent managed](elastic-agent-managed) | Elastic Agent managed by Fleet setup
 [Elastic Agent standalone](elastic-agent-standalone) | Standalone Elastic Agent setup
+
+> Note: The kube-state-metrics (KSM) simple service is not installed as part of the above manifests and needs to be installed from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics).
+
+## Kustomize Templates
+
+In addition to the above manifests, the list below includes the official [kustomize](https://github.com/kubernetes-sigs/kustomize) templates to run them in kubernetes:
+
+Agent Scenario | Description
+---- | ----
+[Elastic Agent managed - Default ](./default/elastic-agent-managed/) | Default Elastic Agent managed by Fleet setup. Includes the installation of kube-state-metrics (KSM) in default configuration.
+[Elastic Agent standalone Default ](./default/elastic-agent-standalone/) | Default Standalone Elastic Agent setup
+[Elastic Agent managed - With KSM in autosharding configuration ](./ksm-autosharding/elastic-agent-managed/) | Elastic Agent managed by Fleet setup with [KSM in autosharding configuration](https://github.com/kubernetes/kube-state-metrics#automated-sharding)
+[Elastic Agent standalone - With KSM in autosharding configuration](./ksm-autosharding/elastic-agent-standalone/) | Standalone Elastic Agent setup with [KSM in autosharding configuration](https://github.com/kubernetes/kube-state-metrics#automated-sharding)
+
```

Comment:
We need to mention here briefly what each scenario is for or just link to the section below where autosharding mode is explained.

Reply:
Updated, how about now?